niceman/resource/base.py

```diff
@@ -11,6 +11,10 @@
 import attr
 from importlib import import_module
 import abc
+try:
+    from ConfigParser import NoSectionError
+except:
+    from configparser import NoSectionError
```

Comment:
in such cases, just check if six does not provide needed "moves", in this case it does:
```from six.moves.configparser import NoSectionError```